lnwallet/chanfunding/wallet_assembler.go

```diff
@@ -289,9 +289,10 @@ func (w *WalletAssembler) ProvisionChannel(r *Request) (Intent, error) {
 		// Otherwise do a normal coin selection where we target a given
 		// funding amount.
 		default:
+			dustLimit := w.cfg.DustLimit
 			localContributionAmt = r.LocalAmt
 			selectedCoins, changeAmt, err = CoinSelect(
-				r.FeeRate, r.LocalAmt, coins,
+				r.FeeRate, r.LocalAmt, dustLimit, coins,
```

Comment:
We're still checking `changeAmt > dustLimit` below this, could remove that check or add a sanity check before we add the change output? Since we're now checking dust in `CoinSelect`.